source/api_documentation/network_integration/js_tags/js_revision_parameters.rst

```diff
@@ -0,0 +1,168 @@
+
+  * - auto_create_local_ringpools
+    - boolean (default false)
+    - Auto create a ring pool with local numbers
+
+  * - auto_create_toll_free_ringpools
+    - boolean (default false)
+    - Auto create a ring pool with toll free numbers
+
+  * - auto_swap
+    - boolean (default false)
+    - Set to true if you want the Invoca JS to automatically detect and replace all numbers on page. This can be useful if you have a large list of phone numbers to manage, or don't know all of the numbers listed across your site. If true, `number_selector` and `number_to_replace` are not needed.
+
+  * - body
+    - string
+    - Response body of the JS Tag to be added to the site
+
+  * - cache_lifetime_days
+    - integer (default 30)
+    - This setting determines how long Invoca identifies a visitor to your landing page as a unique visitor. By default, your Tag's attribution window is set to 30 days.
+
+  * - custom_code
+    - string
+    - The custom code is a scoped function to insert any custom JavaScript to manipulate the options config that will be passed into the Invoca Integration.
+
+  * - **custom_data_fields**
+    - array of objects
+    - Marketing data fields
+
+  * - custom_data_fields[].enabled
+    - boolean
+    - Status of the marketing data field.
+
+  * - custom_data_fields[].partner_name
+    - string
+    - Marketing data field name.
+
+  * - custom_data_fields[].friendly_name
+    - string
+    - Marketing data field friendly name.
+
+  * - custom_data_fields[].attribution_model
+    - string
+    - Marketing data field model type.
+
+  * - custom_data_fields[].data_source_type
+    - string
+    - Data source for the marketing data field.
+
+  * - custom_data_fields[].data_source_name
+    - string
+    - Friendly name for the marketing data field.
+
+  * - default_campaign
+    - object
+    - Default campaign for the all numbers found except for ones in `numbers_to_replace` and `numbers_to_not_replace` configurations.
+
+  * - default_campaign.default_campaign_id
+    - integer
+    - Internal campaign id of the default campaign.
+
+      TBD: SHOULD THIS BE UPDATED TO advertiser_campaign_id_from_network.
+
+  * - default_campaign.default_campaign_name
+    - string
+    - Name of the default campaign
+
+  * - destination_param
+    - string
+    - Destination key for dynamic routing.
+
+  * - message
+    - string
+    - Auto-generated or user defined note for the revision.
+
+  * - **numbers_to_replace**
+    - array of objects
+    - Specify which number(s) to replace and what campaign ID to use for each number.
+
+  * - numbers_to_replace[].phone_number
+    - string
+    - Phone number to be replaced.
+
+  * - numbers_to_replace[].phone_number_e164
+    - string
+    - E.164 version of the phone number to be replaced.
+
+  * - numbers_to_replace[].advertiser_campaign_id
+    - integer
+    - Id of the advertiser campaign to be used for the above phone number.
+
+      TBD: REMOVE THIS KEY
```

Comment:
Should this be removed in this ticket? I was trying to follow the conversation with Carson but wasn't clear if we were leaving this for the next update.

Reply:
In my reply to the question regarding the `id_from_network` versus the internal `id`, I was thinking that of the two, we should display the external id, since that appears to be the one we operate on in the [advertiser_campaign API](https://developers.invoca.net/en/2019-05-01/api_documentation/network_integration/advertiser_campaigns/index.html) and thus is probably the more valuable of the two.  

I don't think it the worst thing if we include both but I believe they'll only be interested in knowing the id_from_network based on the above campaign documentation.